src/status_im/utils/config.cljs

```diff
@@ -17,7 +17,7 @@
 ;; NOTE(oskarth): Feature flag deprecation lifecycles. We want to make sure
 ;; flags stay up to date and are removed once behavior introduced is stable.
 
-(goog-define INFURA_TOKEN "d3633f237cbd4649a639067d1807584c")
+(goog-define INFURA_TOKEN "f315575765b14720b32382a61a89341a")
```

Comment:
:warning: :warning: :warning: :warning: (or is it a dev token?)

Reply:
oops, removing